core/src/migrations/000084-optionsProfileToRecords.ts

```diff
@@ -0,0 +1,59 @@
+import Sequelize from "sequelize";
```

Comment:
This migration looks good now, but it probably belongs within the salesforce plugin - it only applies to options from salesforce destinations.  Please move this into the plugin and it will be good-to-go!